website/src/components/demos/plot-demo.js

```diff
@@ -90,32 +71,18 @@ export default class PlotDemo extends Component {
     }
   }
 
-  @autobind _onViewportChange(viewport) {
-    this.setState({
-      viewport: {...this.state.viewport, ...viewport}
-    });
-  }
-
   render() {
     const {
       viewport: {width, height},
       params: {resolution, showAxis}
     } = this.props;
-    const {viewport, equation, hoverInfo} = this.state;
-
-    const canvasProps = {
-      ...viewport,
-      width,
-      height
-    };
-    const orbitViewport = OrbitController.getViewport(canvasProps);
+    const {equation, hoverInfo} = this.state;
 
     return (
-      <OrbitController
```

Comment:
can't support user interaction in plot demo anymore?

Reply:
This  is moved into the overlay component.